docs/scenarios_authoring/TestingAndDebugging.md

```diff
@@ -56,6 +56,8 @@ For example, the below records can be used to test if the scenario has exactly o
 {"sourceId":"kafka1","record":{"keySchemaId":null,"valueSchemaId":null,"consumerRecord":{"key":null,"value":{"clientId":"4","amount":30,"eventDate":1674548921},"topic":"transactions","partition":0,"offset":58209,"timestamp":1674548933921,"timestampType":"CreateTime","headers":{},"leaderEpoch":0}},"timestamp":1674548933921}
 {"keySchemaId":null,"valueSchemaId":null,"consumerRecord":{"key":null,"value":{"clientId":"4","amount":30,"eventDate":1674548921},"topic":"transactions","partition":0,"offset":58209,"timestamp":1674548933921,"timestampType":"CreateTime","headers":{},"leaderEpoch":0}}
 ```
+The `sourceId` field contains name of the source node as specified in the scenario and the `record` field contains json representation of the event. 
```

Comment:
1. "contains name of the source node" -> "contains the name of the source node"
2. "contains json representation of the event." -> "contains the JSON representation of the event."
3. "the scenario and the `record` field" -> "the scenario, and the `record` field"